app/models/seller_extra_info.rb

```diff
@@ -0,0 +1,3 @@
+class SellerExtraInfo < ApplicationRecord
```

Comment:
Mind adding a verify that founded is greater than like 1800 and <= the current year?

Reply:
or whatever you think is a good floor for merchants

Reply:
Done. Any chance we have a place founded next year/in the future that ends up onboarding onto this? I guess unlikely.

Reply:
Probably unlikely